### PR TITLE
DX-2272 Added cURL Messaging Snippets and Renamed Files

### PR DESCRIPTION
**Pull request by ckoegel**
the only modified files are the `0-curl.sh` files. All others are just renamed

### CODE REVIEW

**Review by ajrice6713 — APPROVED** [review]

File: site/code-snippets/Messaging/GetMedia/Curl.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/media/media-id-123' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/GetMedia/Curl.sh:2]:
```suggestion
```

File: site/code-snippets/Messaging/GetMedia/curl.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/media/media-id-123' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/GetMedia/curl.sh:2]:
```suggestion
```

File: site/code-snippets/Messaging/GetMessages/Curl.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/messages?sourceTn=$BW_NUMBER' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/GetMessages/Curl.sh:2]:
```suggestion
```

File: site/code-snippets/Messaging/GetMessages/curl.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/messages?sourceTn=$BW_NUMBER' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/GetMessages/curl.sh:2]:
```suggestion
```

File: site/code-snippets/Messaging/ListMedia/Curl.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/media' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/ListMedia/Curl.sh:2]:
```suggestion
```

File: site/code-snippets/Messaging/ListMedia/curl.sh [hunk 6]
```diff
@@ -0,0 +1,3 @@
+curl 'https://messaging.bandwidth.com/api/v2/users/$BW_ACCOUNT_ID/media' \
+    -X GET \
```
Comment (hamermike) [inline site/code-snippets/Messaging/ListMedia/curl.sh:2]:
```suggestion
```

File: site/code-snippets/MultiFactorAuth/messagingTwoFactor/curl.sh [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    -H 'Authorization: Basic $cURL_AUTH' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "to": "$USER_NUMBER",
+        "from": "$BW_MFA_NUMBER",
+        "applicationId": "$BW_MFA_MESSAGING_APPLICATION_ID",
+        "scope": "scope",
+        "message": "Your temporary {NAME} {SCOPE} code is {CODE}",
+        "digits": 5
+    }'
```
Comment (hamermike) [inline site/code-snippets/MultiFactorAuth/messagingTwoFactor/curl.sh:12]: Swap to the shorthand.

File: site/code-snippets/PhoneNumberLookup/LookupRequest/curl.sh [hunk 8]
```diff
@@ -0,0 +1,9 @@
+curl --location --request POST 'https://numbers.bandwidth.com/api/v1/accounts/$BW_ACCOUNT_ID/tnlookup' \
+    --header 'Authorization: Basic $cURL_AUTH' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+        "tns":
+        [
+            "19196104420"
+        ]
+    }'
```
Comment (hamermike) [inline site/code-snippets/PhoneNumberLookup/LookupRequest/curl.sh:9]: Swap to the shorthand

File: site/code-snippets/PhoneNumberLookup/LookupRequestStatus/curl.sh [hunk 9]
```diff
@@ -0,0 +1,2 @@
+curl --location --request GET 'https://numbers.bandwidth.com/api/v1/accounts/$BW_ACCOUNT_ID/tnlookup/004223a0-8b17-41b1-bf81-20732adf5590' \
+    --header 'Authorization: Basic $cURL_AUTH'
```
Comment (hamermike) [inline site/code-snippets/PhoneNumberLookup/LookupRequestStatus/curl.sh:2]: Swap to the shorthand.

**Review by hamermike — CHANGES_REQUESTED** [review]

File: site/scripts/env-replace/curl.json [hunk 10]
```diff
@@ -1,13 +1,14 @@
 {
-    "BW_USERNAME": "api-username",
-    "BW_PASSWORD": "pi-password",
+    "$BW_USERNAME": "api-username",
+    "$BW_PASSWORD": "pi-password",
```
Comment (ajrice6713) [inline site/scripts/env-replace/curl.json:3]:
```suggestion
    "$BW_PASSWORD": "api-password",
```

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by hamermike — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**